prow/config.yaml

```diff
@@ -2931,7 +2931,7 @@ periodics:
       - name: GOOGLE_APPLICATION_CREDENTIALS
         value: /etc/service-account/service-account.json
       - name: USER
-        value: prow
+        value: jenkins
```

Comment:
I'm not sure? If it's not used then feel free to change the user name, and remove the bind ssh secret  from the pod spec :-)